ts/src/header-validator/source.test.ts

```diff
@@ -1097,6 +1099,34 @@ const testCases: TestCase[] = [
       },
     ],
   },
+
+  {
+    name: 'max-aggregatable-reports-wrong-type',
+    input: `{
+      "destination": "https://a.test",
+      "max_aggregatable_reports": "2"
```

Comment:
Please also add test cases for:

1. Negative.
2. Zero.
3. JSON number but non-integer.

Reply:
Done.